astropy/stats/funcs.py

```diff
@@ -725,7 +725,57 @@ def bootstrap(data, bootnum=100, samples=None, bootfunc=None):
     Returns
     -------
     boot : numpy.ndarray
-        Bootstrapped data. Each row is a bootstrap resample of the data.
+        Bootstrapped data. Each row is a bootstrap resample of the data. The
+        columns will correspond to the outputs of bootfunc.
+
```

Comment:
Can this be clarified a little bit as this changes depending on whether or not bootfunc is specified.   So something like:  'If bootfunc is None, each row is a bootstrap resample of the data.   If bootfunc is specified, the columns of each row will correspond to the outputs of bootfunc'.